src/meta/src/manager/catalog/mod.rs

```diff
@@ -1253,7 +1252,7 @@ where
             for &dependent_relation_id in &sink.dependent_relations {
                 database_core.increase_ref_count(dependent_relation_id);
             }
-            user_core.increase_ref(sink.owner);
+            user_core.increase_ref_count(sink.owner, 1);
```

Comment:
```suggestion
            user_core.increase_ref(sink.owner);
```
If increasing one ref count, suggest to use `increase_ref` instead of `increase_ref_count`. Otherwise, you should add a comment and explain what "1" means.